clients/gitlabrepo/client.go

```diff
@@ -228,6 +229,22 @@ func (client *Client) Close() error {
 	return nil
 }
 
+func CreateGitlabDotComClient(ctx context.Context) (clients.RepoClient, error) {
+	var err error
+	var gitlabComRepo clients.Repo
+	if gitlabComRepo, err = MakeGitlabRepo("https://gitlab.com/gitlab-org/gitlab"); err != nil {
```

Comment:
Is there significance to this repo? Or is this just an arbitrary repo on the gitlab.com host?

Reply:
It's arbitrary; changed this to use `gitlab.WithBaseURL`